src/objective-c/tests/GRPCClientTests.m

```diff
@@ -109,13 +109,13 @@ - (void)setUp {
   [GRPCCall useInsecureConnectionsForHost:kHostAddress];
 
   // This method isn't implemented by the remote server.
-  kInexistentMethod = [[ProtoMethod alloc] initWithPackage:kPackage
+  kInexistentMethod = [[GRPCProtoMethod alloc] initWithPackage:kPackage
                                                    service:kService
                                                     method:@"Inexistent"];
-  kEmptyCallMethod = [[ProtoMethod alloc] initWithPackage:kPackage
+  kEmptyCallMethod = [[GRPCProtoMethod alloc] initWithPackage:kPackage
                                                   service:kService
                                                    method:@"EmptyCall"];
-  kUnaryCallMethod = [[ProtoMethod alloc] initWithPackage:kPackage
+  kUnaryCallMethod = [[GRPCProtoMethod alloc] initWithPackage:kPackage
                                                   service:kService
                                                    method:@"UnaryCall"];
```

Comment:
Please realign the colons in these lines (and the ones above). You can have XCode do it by selecting them and hitting `ctrl + i`.